### PR TITLE
Draft of Unit Testing Redux

### PR DESCRIPTION
**Pull request by joehandlebar**
I've never written about code so definitely needs love.
I mostly referenced the ng2-redux-starter. :computer: 

there's a PR w/ the examples here:
https://github.com/rangle/angular2-redux-starter/pull/34

### CODE REVIEW

File: handout/testing/unit_testing_redux.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
+# Unit Testing Redux
+
+The units that make up redux are actions and reducers. Lets try testing them.
```
Comment (bennett000) [inline handout/testing/unit_testing_redux.md]: Maybe something more expansive, like:

Unit testing Redux is a very straightforward process.  There are two units, reducers, and actions.  Reducers are pure functions that lend themselves well to testing.  Actions trigger changes in a Redux system, there are two broad categories of actions, synchronous, and asynchronous.  Synchronous actions are quite straightforward to test, but asynchronous actions are slightly more involved.  The examples below should provide the reader with a strong foundation for testing Redux applications.

File: handout/testing/unit_testing_redux.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+        type: DECREMENT_COUNTER                                 
+      });                                                       
+  });                                                           
+
+  ...
+
+});                                                             
+```
+
+We just make sure that our action creators do indeed create actions.
```
Comment (bennett000) [inline handout/testing/unit_testing_redux.md]: We should probably make sure that actions are actions, as in that they are POJO's with a type property.

File: handout/testing/unit_testing_redux.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+      };
+    }                                                
+  });                                                
+
+  // some thunk that dispatches our expectedActions
+  store.dispatch(CounterActions.zigZagZigala());   
+});
+```
+
+There's more than one flavor of asynchronous action creating in redux, so you might have to cook something that fits your unit's needs.
```
Comment (bennett000) [inline handout/testing/unit_testing_redux.md]: This sentence could be more formal.  Something like:

These asynchronous examples are based on the `thunk` pattern.  Redux is very flexible and there are many other ways to implement asynchronous actions, `thunk`s are merely the most common.  Other asynchronous implementations may require a different approach to unit testing.

**Comment by e-schultz** [comment]
Closing to address final edits/etc in #93

**Pull request closed without merging**